questionnaires/esm/esm_armt.json

```diff
@@ -870,11 +870,11 @@
             },
             {
                 "code": "3",
-                "label": "Passive leisure "
+                "label": "Passive leisure (TV, computer, videogames, reading) "
             },
             {
                 "code": "4",
-                "label": "Active leisure "
+                "label": "Active leisure (walking, sports, gardening, going out) "
```

Comment:
Has this been tested in the UI? The extra info was removed because it was going out of bounds of the button in UI.

Reply:
I think this was caused by my fixing of the merge conflicts when I tried to merge master into test. I’ll change this back.